sdk/examples/wallet/spammer.rs

```diff
@@ -57,22 +57,23 @@ async fn main() -> Result<()> {
         .with_coin_type(SHIMMER_COIN_TYPE)
         .finish()
         .await?;
-
-    // Ensure there's some base coin balance
     let account = get_or_create_account(&wallet, ACCOUNT_ALIAS).await?;
+
+    // Ensure there are enough addresses in the account.
     let addresses = ensure_enough_addresses(&account, num_simultaneous_txs).await?;
```

Comment:
There is no point in having multiple addresses

Reply:
changed